packages/init/src/utils/create-repo.ts

```diff
@@ -1,30 +1,22 @@
 import type { Models } from "@slicemachine/core";
 import * as logs from "./logs";
-import * as Prismic from "@slicemachine/core/build/prismic";
+import { Client } from "./client";
 
 export function createRepository(
```

Comment:
We could but since it's a singleton, it doesn't really make sense.
That would just be adding more parameters when it already is accessible.

It could make sense to make sure it is instantiated, I wanted to avoid passing it everywhere.
On top of that, the singleton approach allowed me to create more calls that are required for this application based on the Client's code.

Reply:
nice